### PR TITLE
Re-enable ApiScan

### PR DESCRIPTION
**Pull request by delmyers**
Updates ApiScan job to use new build machines.

Note, this PR is purposefully against the release branch because we need to run our compliance builds before releasing new binaries.

### CODE REVIEW

File: build/yaml/jobs/codeanalysis/apiscan.yaml [hunk 1]
```diff
@@ -25,13 +25,20 @@ jobs:
   - checkout: self
     clean: true
 
+  # Expire ApiScan one day before declared expirary, to make sure that we update it before
+  # it expires, regardless of timezone
+  - powershell: if ((Get-Date).Add((New-TimeSpan -Days 1)) -ge (Get-Date -Date "$(ApiScanExpiration)")) { throw "ApiScan key expired. Please update" }
```
Comment (WilliamXieMSFT) [inline build/yaml/jobs/codeanalysis/apiscan.yaml:30]: >( [](http://example.com/codeflow?start=35&length=1)

nit extra ( )

**Review by WilliamXieMSFT — APPROVED** [review]
:shipit:

**Review by jander-msft — APPROVED** [review]

**Pull request merged**